src/llfs/storage_context.cpp

```diff
@@ -94,6 +94,41 @@ Status StorageContext::add_existing_file(const batt::SharedPtr<StorageFile>& fil
   return OkStatus();
 }
 
+//==#==========+==+=+=++=+++++++++++-+-+--+----- --- -- -  -  -   -
+//
+StatusOr<std::unique_ptr<LogDeviceFactory>> StorageContext::recover_log_device(
+    const boost::uuids::uuid& uuid, const LogDeviceRuntimeOptions& log_runtime_options)
+{
+  batt::SharedPtr<StorageObjectInfo> info = this->find_object_by_uuid(uuid);
+  if (!info) {
+    return {batt::StatusCode::kNotFound};
+  }
+
+  switch (info->p_config_slot->tag) {
+      //----- --- -- -  -  -   -
+    case PackedConfigSlotBase::Tag::kLogDevice:
```

Comment:
Just noting a spot that might break once we switch to full using LogDevice2.

Reply:
Agree.  Is there a change you'd like to see here?